src/main/java/com/nooul/apihelpers/springbootrest/utils/QueryParamExtractor.java

```diff
@@ -1,57 +1,70 @@
+/*
+ * https://github.com/Nooul/spring-boot-rest-api-helpers
+ * 
+ * Released under the MIT License.
+ * Please refer to LICENSE file for licensing information.
+ */
+
 package com.nooul.apihelpers.springbootrest.utils;
 
+import com.github.openjson.JSONArray;
+import com.github.openjson.JSONObject;
+import com.github.openjson.JSONTokener;
 import com.nooul.apihelpers.springbootrest.entities.QueryParamWrapper;
-import org.apache.commons.lang3.StringUtils;
-import org.json.JSONArray;
-import org.json.JSONObject;
-import org.json.JSONTokener;
 
 import java.io.UnsupportedEncodingException;
 import java.net.URLDecoder;
 
+/**
+ * Query parameters extractor
+ */
 public class QueryParamExtractor {
 
+    /**
+     * Extract query parameters
+     * 
+     * @param filterStr
+     * @param rangeStr
+     * @param sortStr
+     * @return
+     */
     public static QueryParamWrapper extract(String filterStr, String rangeStr, String sortStr) {
 
-
-
-        Object filterJsonOrArray;
+        // validate filter string
         if (StringUtils.isBlank(filterStr)) {
             filterStr = "{}";
         }
-
-        //https://stackoverflow.com/a/18368345
```

Comment:
these comments are needed for context since they are copy pasted from stackoverflow - I will readd them

Reply:
Got it. I prefer to clean those comment our cause one I add code, rarely the stackoverflow source change code.

Reply:
not sure I think it is a requirement from stackoverflow plus helps other developers to have a context when it is a cryptic thing